src/components/Camera/PermissionsPrimer/style.css

```diff
@@ -0,0 +1,71 @@
+@import (less) "../../Theme/constants.css";
+
+.text {
+  margin-top: 64px;
+  margin-bottom: 80px;
+}
+
+.image {
+  border-radius: 3px;
+  border: 1px solid #D8D8D8;
+  font-size: 14px;
+  margin: 30px 0 8px;
+}
+
+.image p {
+  margin: 16px;
+}
+
+.reasons {
+  text-align: center;
+  font-size: 14px;
+  min-height: 3em;
+}
+
+.reason {
+  margin-top: 8px;
+}
+
+.graphic {
+  border: 1px solid #DFE0E0;
+  border-radius: 5px;
+  background-color: #fff;
+  box-shadow: 0 3px 6px #D8D8D8;
+  background-image: url('assets/allow.svg');
+  background-repeat: no-repeat;
+  background-size: 100%;
+  height: 121px;
+  margin: 0 auto 30px;
+  position: relative;
+  width: 270px;
+
+  @media (--small-viewport) {
+    height: 100px;
+    width: 224px;
+  }
+}
+
+.allow {
+  top: 70%;
+  position: absolute;
+  right: 0;
+  text-align: center;
+  width: 50%;
+  font-weight: 600;
+
+  @media (--small-viewport) {
+    top: 65%;
+  }
+}
+
+.allow::after {
+  content: '';
+  border: 4px solid #00C9A1;
+  width: 80px;
+  height: 80px;
+  border-radius: 100%;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate3d(-50%, -50%, 0);
```

Comment:
how reliable is this centring on ie 11?
Also does it need to be `translate3d`? why not just `translate`?

Reply:
It's reliable on IE11, and it doesn't have to be `translate3d`. Thanks!

Reply:
Just tested it on IE11 to be sure, and looks good